windows/ReactTestApp/MainPage.cpp

```diff
@@ -82,13 +88,37 @@ namespace winrt::ReactTestApp::implementation
                 AppTitle().Text(to_hstring(manifest.value().displayName));
             }
 
+            auto keyboardAcceleratorKey = VirtualKey::Number1;
```

Comment:
It seems like Windows uses `Ctrl+Shift+0` for some input language settings, so this keyboard event doesn't reach the application. It can be bypassed by changing language settings, but we should probably not force users to do that.    
Combinations of `Ctrl`, `Shift` and `1-9` work fine.

Reply:
We should change macOS as well to make it consistent.

Reply:
https://github.com/microsoft/react-native-test-app/pull/195